ui/desktop/app/helpers/group-by.js

```diff
@@ -0,0 +1,17 @@
+import { helper } from '@ember/component/helper';
+
+const groupByReducer = (items, key) => {
```

Comment:
Can we add a comment/documentation on what this function does?  similar to the `is-loading` helper

Reply:
Done.